butterknife-compiler/src/main/java/butterknife/compiler/BindingClass.java

```diff
@@ -730,6 +734,24 @@ private boolean hasMethodBindings() {
     return false;
   }
 
+  private boolean methodBindingsNeedFinder() {
+    for (ViewBindings viewBindings : viewIdMap.values()) {
+      for (Map<ListenerMethod, Set<MethodViewBinding>> listenerMethodSetMap :
+          viewBindings.getMethodBindings().values()) {
+        for (Set<MethodViewBinding> methodViewBindings : listenerMethodSetMap.values()) {
+          for (MethodViewBinding methodViewBinding : methodViewBindings) {
+            for (Parameter parameter : methodViewBinding.getParameters()) {
+              if (!parameter.getType().toString().equals(VIEW.toString())) {
```

Comment:
Yeah, I probably should have looked at what that was doing. Saw you pushed a commit here: https://github.com/JakeWharton/butterknife/commit/f8d673d1724a330294a69d0325d6ca2c075a7edf feel free to just take that instead.